controllers/akodeploymentconfig/akodeploymentconfig_controller_intg_test.go

```diff
@@ -369,27 +393,22 @@ func intgTestAkoDeploymentConfigController() {
 		BeforeEach(func() {
 			createObjects(akoDeploymentConfig, cluster, controllerCredentials, controllerCA)
 			conditions.MarkTrue(cluster, clusterv1.ReadyCondition)
-			err = ctx.Client.Status().Update(ctx, cluster)
-			Expect(err).To(BeNil())
+			Eventually(func() error {
+				err = ctx.Client.Status().Update(ctx, cluster)
+				return err
+			}).Should(BeNil())
```

Comment:
Unsure if this is necessary as well?

Reply:
Good catch, forgot to remove object in `Update` from the global search